moloch_connector.py

```diff
@@ -245,7 +245,7 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
         # Create a URL to connect to
         try:
             url = '{url}{endpoint}'.format(url=self._server_url, endpoint=endpoint)
-        except Exception as e:
+        except:
```

Comment:
Don't we want to have "except Exception" here?

Reply:
Hi Tony - Exception e is not used anywhere, which is detected by somgrep.

Reply:
Oh, what I meant was not using a bare exception. Instead of "except Exception as e", just "except Exception". I only mentioned it because @rgil-splunk mentioned it to me before in one of his reviews of my code. I believe the bare except would catch certain exceptions we wouldn't want to catch such as KeyboardInterrupt and SystemExit. I don't know if this would ultimately cause an issue though, perhaps Ronald can comment?

Reply:
what you mentioned Tony pretty much covers it :)

while there is a low probability of it causing an issue, it would be difficult to debug and pinpoint what happened if there is an issue given it would happen during failure / exit states

link for reference: https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except/54948581#54948581

Reply:
Make sense! Thanks Ronald and Tony. Let me fix this